ibc-apps/ics721-nft-transfer/types/Cargo.toml

```
@@ -20,6 +20,7 @@ all-features = true
 [dependencies]
 # external dependencies
 borsh           = { workspace = true, optional = true }
+base64          = "0.21.6"
```

```suggestion
base64          = { version = "0.21.6", default-features = false }
```

And add the `base64/std` under the `std` feature.